heron/common/src/python/handler/access/heron.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
I reverted this file and got it to work. For this patch can we revert this file and then try to improve it in another patch?